packages/veritone-react-common/src/components/FormBuilder/FormItems/Radio.js

```diff
@@ -24,11 +24,14 @@ export default function Radio({
     (e) => onChange({ name, value: e.target.value }),
     [name, onChange, value]
   );
+
+  const styles = useStyles({});
```

Comment:
Remove if not used

Reply:
Actually we are using it, so we can not remove it